### PR TITLE
Release version 21.13.2

### PR DESCRIPTION
**Pull request by SamJamCul**
* Change tags and styling for feedback component ([PR #1207](https://github.com/alphagov/govuk_publishing_components/pull/1207))

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -317,5 +317,8 @@ DEPENDENCIES
   webmock (~> 3.6.0)
   yard
 
+RUBY VERSION
```
Comment (kr8n3r) [inline Gemfile.lock]: can you remove this please?

Comment (SamJamCul) [reply Gemfile.lock]: Done!

Comment (kr8n3r) [reply Gemfile.lock]: 💥

**Comment by alex-ju** [comment]
It looks like this release is empty. Sam's PR #1207 was released as part of 21.13.1 looking at the git history (although is [not captured in the PR description](https://github.com/alphagov/govuk_publishing_components/pull/1208))

**Pull request merged**